src/Core/Models/Business/OrganizationLicense.cs

```diff
@@ -131,10 +133,10 @@ public OrganizationLicense()
     /// <summary>
     /// Represents the current version of the license format. Should be updated whenever new fields are added.
     /// </summary>
-    private const int CURRENT_LICENSE_FILE_VERSION = 9;
```

Comment:
I updated this to 11 although I'm not sure why it was still at 9 when there were already checks for version 10.

Reply:
Changed value to 10.